components/Projects/ProjectPage/TechList/index.tsx

```diff
@@ -1,30 +1,34 @@
 /** @jsx jsx */
 import { jsx } from '@emotion/core'
 import { useTheme } from 'emotion-theming'
+import BrandIcon from 'components/Elements/BrandIcons'
+import type { Brand } from 'components/Elements/BrandIcons'
```

Comment:
cant this be combined with the above import?

Reply:
No